testsuite/test_api.py

```diff
@@ -19,7 +22,12 @@ def __init__(self, client):
         self.client = client
 
     def start(self):
-        resp = self.client.post('/v0/jobs')
+        credentials = base64.b64encode(b'test:test').decode('utf8')
+        resp = self.client.get('/v0/token', **{'HTTP_AUTHORIZATION': 'Basic ' + credentials}).json()
```

Comment:
You can save a few chars by just writing `resp = self.client.get('/v0/token', HTTP_AUTHORIZATION=f'Basic {credentials}').json()`

Reply:
Also please use a different variable name, e.g. `auth_resp`, makes reading the code easier.